modules/library/src/main/scala/zio/elasticsearch/ElasticQuery.scala

```diff
@@ -181,11 +182,12 @@ object ElasticQuery {
     override def toJson: Option[(String, Json)] = None
   }
 
-  private[elasticsearch] final case class RangeQuery[A, LB <: LowerBound, UB <: UpperBound] private (
```

Comment:
Currently, we are working on some code improvements - take a look [here](https://github.com/lambdaworks/zio-elasticsearch/pull/82). 

Therefore, after that is applied we should be able to keep `private` here.

Reply:
Changes done !